src/exports.ts

```diff
@@ -168,15 +256,21 @@ function parseExport(
  * pkg.main and pkg.module will be added to ['.'] if exists
  */
 
-export function getExportPaths(pkg: PackageMetadata) {
+export function getExportPaths(pkg: PackageMetadata, cwd: string) {
   const pathsMap: Record<string, FullExportCondition> = {}
   const packageType = getPackageType(pkg)
   const isCjsPackage = packageType === 'commonjs'
   
   const { exports: exportsConditions } = pkg
-  
+
   if (exportsConditions) {
-    const paths = parseExport(exportsConditions, packageType)
+    let resolvedExportsConditions = exportsConditions
+
+    if (Object.keys(exportsConditions).some((key) => key.includes('*'))) {
```

Comment:
The nodejs exports supports various cases like: `./abc/*`, `./*`, `./dist/*/lib`, I think for bundled case we should only support some very restricted case like only `./*, but not others. As we also only traverse the 1st layer of the `src/` folder.